psl/psl/tests/attributes/composite_index.rs

```diff
@@ -527,7 +527,7 @@ fn pointing_to_a_non_existing_type() {
     let error = parse_unwrap_err(&dml);
 
     let expected = expect![[r#"
-        [1;91merror[0m: [1mType "C" is neither a built-in type, nor refers to another model, custom type, or enum.[0m
+        [1;91merror[0m: [1mType "C" is neither a built-in type, nor refers to another model, custom type, or enum. Did you mean ..."c"[0m
```

Comment:
`c` does not exist in this test, so this would be a bad suggestion.
I think the issue suggeted to only suggest alternative spelling if those _actually exist in the datamodel_.

Reply:
Yeah, you are right, I will work on that, do you think the modifications on error.rs is the way to go?

I was trying to find if there is any variable or function that gives info about the data model, or does that need to be implemented?

Reply:
Sorry, I am the wrong person to ask about that. I only could connect that the modified error message in the test would probably not match what the issue suggested 😆